proj/dse_template/README.md

```diff
@@ -0,0 +1,61 @@
+# Design Space Exploration in CFU Playground
+This project template is intended for performing architectural design space exploration (DSE) in CFU Playground. 
+
+## Prerequisites & Setup
+1. Playground Installation
+- Make sure you have run `$CFU_ROOT/scripts/setup`
+- Make sure you have installed and entered the Symbiflow environment via `make install-sf` & `make enter-sf`
+- Test this with `which symbiflow_synth`
+
+2. Install the VexRiscv Build Tools
+- Run `$CFU_ROOT/scripts/setup_vexriscv_build.sh`
+
+3. Install Vizier 
+- Run `$CFU_ROOT/scripts/setup_vizier.sh`
+
+4. Test DSE 
+- In `proj/dse_template` run `python vizier_dse.py`
+- Sample output should look like this (with different raw values):
```

Comment:
Can you add a note about how long it will take to get to this point?

Reply:
@tcal-x added :)